keyboards/keebio/viterbi/keymaps/vosechu/keymap.c

```diff
@@ -0,0 +1,116 @@
+),
+[TT1] = LAYOUT_ortho_half_5x7( // Override WASD with arrows
+  _______ , _______ , _______ , _______ , _______ , _______ , _______ ,
+  _______ , _______ , KC_UP   , _______ , _______ , _______ , _______ ,
+  _______ , KC_LEFT , KC_DOWN , KC_RGHT , _______ , _______ , _______ ,
+  _______ , _______ , _______ , _______ , _______ , _______ , _______ ,
+  _______ , _______ , _______ , _______ , _______ , _______ , _______
+),
+[TT2] = LAYOUT_ortho_half_5x7( // Browser layer
+  C(KC_W) , C(KC_1) , C(KC_T) , C(KC_9) , _______ , _______ , _______ ,
+  WBWSRBK , WTABLFT , KC_UP   , WTABRGT , WBWSRFW , _______ , _______ ,
+  KC_ESC  , KC_LEFT , KC_DOWN , KC_RGHT , C(KC_R) , _______ , _______ ,
+  _______ , _______ , _______ , _______ , _______ , _______ , TT(T23X),
+  _______ , _______ , _______ , _______ , _______ , _______ , _______
+),
+[TT3] = LAYOUT_ortho_half_5x7( // OS X override layer
+  _______ , _______ , _______ , _______ , _______ , _______ , _______ ,
+  _______ , _______ , _______ , _______ , _______ , _______ , _______ ,
+  _______ , _______ , _______ , _______ , _______ , _______ , TT(T23X),
+  _______ , _______ , _______ , _______ , _______ , _______ , _______ ,
+  _______ , _______ , _______ , KC_LGUI , _______ , _______ , _______
+),
+[T23X] = LAYOUT_ortho_half_5x7( // Browser layer
+  G(KC_W) , G(KC_1) , G(KC_T) , G(KC_9) , _______ , SLACKTB , _______ ,
+  BWSR_BK , TAB_LFT , KC_UP   , TAB_RGT , BWSR_FW , SLACKUP , _______ ,
+  KC_ESC  , KC_LEFT , KC_DOWN , KC_RGHT , G(KC_R) , SLACKDN , _______ ,
+  _______ , _______ , KC_Q    , KC_J    , KC_K    , _______ , _______ ,
+  _______ , _______ , _______ , KC_LGUI , _______ , _______ , _______
+),
+// [_EMPTY] = LAYOUT(
+//   _______ , _______ , _______ , _______ , _______ , _______ , _______ ,
+//   _______ , _______ , _______ , _______ , _______ , _______ , _______ ,
+//   _______ , _______ , _______ , _______ , _______ , _______ , _______ ,
+//   _______ , _______ , _______ , _______ , _______ , _______ , _______ ,
+//   _______ , _______ , _______ , _______ , _______ , _______ , _______
+// ),
+};
+
+void keyboard_post_init_user(void) {
+    // Call the post init code.
+    rgblight_enable_noeeprom(); // enables Rgb, without saving settings
+    rgblight_mode_noeeprom(0);
+    rgblight_sethsv_noeeprom(0, 0, 0);
+}
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+  switch (get_highest_layer(state)) {
+  case TT1:
+    rgblight_sethsv_noeeprom(HSV_BLUE);
+    break;
+  case TT2:
+    rgblight_sethsv_noeeprom(HSV_PURPLE);
+    break;
+  case T23X:
+    rgblight_sethsv_noeeprom(HSV_GOLD);
+    break;
+  case TT3:
+    rgblight_sethsv_noeeprom(HSV_GREEN);
+    break;
+  default: //  for any other layers, or the default layer
+    rgblight_sethsv_noeeprom(0, 0, 0);
+    break;
+  }
+
+  return state;
+}
+
+void suspend_power_down_user(void) {
+    rgblight_disable();
+}
+
+void suspend_wakeup_init_user(void) {
+    rgblight_enable();
+}
```

Comment:
```suggestion
```

You can define `RGBLIGHT_SLEEP` to do this, actually.

Reply:
Awesome! Thank you! I'll fix this.